mi/dataset/driver/ctdav_n/auv/resource/ctdav_n_auv_recov_21.yml

```diff
@@ -1,3 +1,4 @@
+# THIS FILE IS DEPRECATED, REPLACED BY mi/dataset/driver/ctdav_n/auv/resource/ctdav_n_auv_21.yml
```

Comment:
How does an input file become deprecated? If this was a valid test file before, it should still work with the new parser, no? The only way it would be deprecated is if we are no longer supporting an old format, yes?

Reply:
There is no difference between telemetered and recovered data.  So the file name needed to change and the particle_object and the particle_type needed change.  So, my options were:
    1) Create a new yml file with the proper name and remove the 2 original yml files.
    2) Modify one of the original yml files and rename it and remove the other original yml file.
The first option seemed cleaner and to "remove" the olds files, I chose to "deprecate" them, but perhaps that's not the right thing to do, since they are not being referenced in the test method anymore, the new file is.  Should we simply delete them from the repository?